numpy/random/mtrand/mtrand.pyx

```diff
@@ -3426,7 +3426,8 @@ cdef class RandomState:
         Returns
         -------
         samples : ndarray or scalar
-                  where the values are all integers in  [0, n].
+            Samples from logistic distribution, shaped according to
+            `size`. Otherwise, a single value is returned.
```

Comment:
Indeed, that is true.  I think the documentation needs to be more standardized IMO, so I'll go through the doc strings for these broadcasting functions to make sure that that they properly explain how the size of the output varies depending on '''size'' and the size of the inputs.